src/sass/_vars.sass

```diff
@@ -130,8 +133,10 @@ $spacing8: 8.5em
 $break0: 0.0625rem // 1px
 $breakSm: 21rem // 336px
 $breakMdSm: 28rem // 448px
+$breakMdSmSm: 37.5rem // 600px
 $breakMd: 50rem // 768px
 $breakMdLg: 60rem // 960px
+$breakMdMdLg: 64rem // 1024px
```

Comment:
Do we need to add new breakpoints here? Having too many can get to be a bit unwieldy, so I just wonder if we can accomplish what's needed with the existing ones.

Reply:
I added two more because those were what we previously used and it worked. I figured to not try and fix what's working and that it would be easier just to add those two.

Reply:
But I just tested and it works fine with the other breakpoints, so we're good. I've committed the changes.